modules/rest-api/pages/rest-cluster-joinnode.adoc

```diff
@@ -99,13 +99,9 @@ curl -u Administrator:password -d 'clusterMemberHostIp=192.168.0.1' \
 403 Forbidden bad credentials - invalid credentials
 ----
 
-Note that if the node certificate on the new node does not correctly specify the node-name as a Subject Alternative Name, one of the following occurs:
+Note that if the node certificate on the new node does not correctly specify the node-name as a Subject Alternative Name, failure may occur, with a message of the following form: `Attention: Prepare join failed. Failed to establish TLS connection to 10.144.231.102:18091. TLS handshake failure (TLS client: In state wait_cert_cr at ssl_handshake.erl:2084 generated CLIENT ALERT: Fatal - Handshake Failure {bad_cert,hostname_check_failed})`.
```

Comment:
Please change the message. See my comment above.

Reply:
Ok, have changed.

Reply:
Ok, have changed.